.github/workflows/python_ci.yml

```
@@ -4,12 +4,12 @@ jobs:
   python_run_scripts:
     strategy:
       matrix:
-        version: [3.9, 3.10, 3.11]
-    runs-on: ubuntu-latest
+        python: [3.9.12, 3.10.8, 3.11.10]
```

Are the bug fix version numbers required here, or is:
```suggestion
        python: [3.9, 3.10, 3.11]
```
sufficient?